pepper-apis/src/main/java/org/broadinstitute/ddp/route/PostMedicalProviderRoute.java

```diff
@@ -42,74 +46,44 @@ public PostMedicalProviderResponsePayload handle(
         String institutionType = request.params(PathParam.INSTITUTION_TYPE);
         return TransactionWrapper.withTxn(
                 handle -> {
-                    UserDto userDto = handle.attach(JdbiUser.class).findByUserGuid(participantGuid);
-                    if (userDto == null) {
-                        LOG.info("A user with GUID {} was not found [{},{}]", participantGuid, request.pathInfo(), request.ip());
-                        String errMsg = "The user was not found";
-                        throw ResponseUtil.haltError(response, HttpStatus.SC_NOT_FOUND, new ApiError(ErrorCodes.NOT_FOUND, errMsg));
-                    }
+                    UserDto userDto = Optional.ofNullable(handle.attach(JdbiUser.class).findByUserGuid(participantGuid)).orElseGet(
```

Comment:
The old way looks fine to me. Why do we have to wrap it up in an `Optional`? And if we're really going with this, I'll recommend using `orElseThrow()` instead.